interfaces/builtin/gpio_chardev.go

```diff
@@ -203,11 +203,12 @@ func (iface *gpioChardevInterface) UDevConnectedPlug(spec *udev.Specification, p
 func init() {
 	registerIface(&gpioChardevInterface{
 		commonInterface{
-			name:                     "gpio-chardev",
-			summary:                  gpioChardevSummary,
-			baseDeclarationSlots:     gpioChardevBaseDeclarationSlots,
-			connectedSlotKModModules: gpioChardevConnectedSlotKmod,
-			serviceSnippets:          gpioChardevPlugServiceSnippets,
+			name:                         "gpio-chardev",
+			summary:                      gpioChardevSummary,
+			baseDeclarationSlots:         gpioChardevBaseDeclarationSlots,
+			connectedSlotKModModules:     gpioChardevConnectedSlotKmod,
+			serviceSnippets:              gpioChardevPlugServiceSnippets,
+			exclusiveConnectedInterfaces: []string{"gpio"},
```

Comment:
exclusive doesn't feel right, or at least I find it a bit confusing. Maybe it should be named conflictingConnectedInterfaces? to be more in line with how e.g. packages declare conflicts with other packages (`Conflicts: foo bar`)